BUILDING.md

```diff
@@ -45,11 +45,11 @@ Run with **Administrator** privileges:
 choco install make cmake mingw wget
 ```
 
-#### Install Go 1.22
+#### Install Go 1.23
 
-Download and install Go 1.22 from the [official website](https://go.dev/dl/).
+Download and install Go 1.23 from the [official website](https://go.dev/dl/).
 
-> ⚠️ Note: 1.22 version is a strict requirement for now. Version 1.23 won't work
+> ⚠️ Note: 1.23 version is a strict requirement for now. Version 1.24 won't work
```

Comment:
With this PR, I was able to build against 1.24 as wel... :)

Reply:
Its good to know that you can but I'd like the docs to indicate that devs should stick to the versions we build the app from in CI.